closed/autoconf/generated-configure.sh

```diff
@@ -17182,7 +17182,7 @@ fi
       OPENJ9_PLATFORM_CODE=oa64
       OPENJ9_BUILDSPEC="osx_x86-64"
     else
-      as_fn_error $? "Unsupported OpenJ9 platform ${OPENJDK_BUILD_OS}!" "$LINENO" 5
+      as_fn_error $? "Unsupported OpenJ9 platform ${OPENJDK_BUILD_OS}, contact support team!" "$LINENO" 5
```

Comment:
This change is unexpected given the change to custom-hook.m4.

Reply:
Agreed. We'll need to regen the generated-configure.sh for jdk9, jdk10, and fix jdk8 manually. Hoping to get that done within a few hours.